pageserver/src/tenant/timeline.rs

```diff
@@ -867,6 +867,11 @@ impl Timeline {
             .map(|ancestor| ancestor.timeline_id)
     }
 
+    /// Get the ancestor's timeline id
```

Comment:
Note this is wrong, from above method..? But yeah, this is another reason why I don't think it makes sense to add getters by default.

Reply:
fixed